db/migrate/20150729170224_add_boxes.rb

```diff
@@ -0,0 +1,9 @@
+class AddBoxes < ActiveRecord::Migration
+  def change
+    create_table :boxes do |t|
+      t.string :month, null: false
+      t.string :completed, default: false
```

Comment:
I was thinking using completed for the box model would help subscribers know when we completed a printing so that they can get excited about the shipment.